1.8hm.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+
+//�������� ���������, ��������� ���������� ������� ��������� � �������.
+
+int nullCounter(int *arr, int len)
+{
+	int counter = 0;
+	for (int i = 0; i < len; i++) 
+	{
+		if (arr[i] == 0)
+			counter++;

```

Comment:
```suggestion
		if (arr[i] == 0)
		{
			counter++;
		}
```
Всегда пиши фигурные скобки для вложенных выражений.
Так и читать проще, и шанса ошибиться меньше.